assembl/static2/js/app/utils/inputHandler.js

```diff
@@ -0,0 +1,12 @@
+/*
+  Utility method to handle input events fed from a component via a onChange.
+  @params component [React] The component in question
+  @params inputEvent [SyntheticEvent] The event that is triggered from the onChange
+  @returns void
+*/
+export default (component, inputEvent) => {
```

Comment:
Is this the right way to bind an input to a state property? Can't we set an attribute on the React element (HTML tag) for this, like in Angular data-model something?